tox.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 120
-ignore = E305,E402,E721,F401,F403,F405,F821,F841
+ignore = E305,E402,E721,F401,F403,F405,F821,F841,F999
```

Comment:
On a brand new flake install on the ubuntu 16.04 in the Windows Subsystem for Linux, I get many of this kind of errors:
`./torch/autograd/variable.py:741:16: F999 Eq may be undefined, or defined from star imports: _functions`
and `./torch/__init__.py:156:21: F999 _C may be undefined, or defined from star imports: torch._C`
It basically raises one for every star import that we use in our codebase.
Should I remove it?